src/Lib/Validate/Strings.php

```diff
@@ -25,4 +30,24 @@ protected function validateString($value, $message = 'Value is not a string.', $
         
         return true;
     }
+
+    /**
+     * Validates that the passed value is a string
+     * @param  mixed  $value
+     * @param  string $message        - Optional custom error message to throw
+     * @param  string $exceptionClass - Optional namespaced (relative to Exception) exception class to throw
+     *
+     * @throws Validation
+     *
+     * @return boolean
+     */
+    protected function validateStringMinimumLength($value, $minimumLength = 1, $message = 'String cannot be shorter than %d.', $exceptionClass = 'InvalidString')
+    {
+        if (strlen(trim($value)) < $minimumLength) {
+            $namespace = Base::buildExceptionNamespace($exceptionClass);
+            throw new $namespace(sprintf($message, $minimumLength));
+        }
+
+        return true;
+    }
```

Comment:
:+1: I think the traits will remain, but will be class-specific. This will mean they are available from compile time with no expensive autoloading, and it'll keep the classes themselves nice and clean.